packages/qwik/src/optimizer/src/plugins/plugin.ts

```diff
@@ -266,6 +267,11 @@ export function createPlugin(optimizerOptions: OptimizerOptions = {}) {
     }
     opts.csr = !!updatedOpts.csr;
 
+    opts.inlineStylesUpToBytes = optimizerOptions.inlineStylesUpToBytes ?? 0;
```

Comment:
We should add a default value here.

Reply:
Maybe having the _option_ to inline is worth more than inlining by default. Especially when an external .css file can be hashed and cached forever.

Reply:
I think we should keep the previous behavior unless it is overwritten...